migration/1685958638251-ProjectEstimatedMatchingView.ts

```diff
@@ -0,0 +1,51 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class ProjectEstimatedMatchingView1685958638251
+  implements MigrationInterface
+{
+  async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `
+      DROP MATERIALIZED VIEW IF EXISTS project_estimated_matching_view;
+      CREATE MATERIALIZED VIEW project_estimated_matching_view AS
+      SELECT
+        donations_by_user."projectId",
+        donations_by_user."qfRoundId",
+        SUM(donations_by_user."valueUsd") as "sumValueUsd",
+        COUNT(*) as "uniqueDonorsCount",
+        SUM(SQRT(donations_by_user."valueUsd")) as "sqrtRootSum"
+      FROM (
+        SELECT
+          "donation"."projectId",
+          "donation"."qfRoundId",
+          SUM("donation"."valueUsd") as "valueUsd"
+        FROM
+          "donation"
+          INNER JOIN "user" ON "user"."id" = "donation"."userId"
+          INNER JOIN "qf_round" ON "qf_round"."id" = "donation"."qfRoundId"
+        WHERE
+          "donation"."status" = 'verified'
+          AND "user"."passportScore" >= "qf_round"."minimumPassportScore"
```

Comment:
It's decided to skip users' passport score in calculating the estimate matching, so its condition must be removed.
Also, we must fill the view for donations of an active qfRound, not all of them.